hiredis/hiredis.pyi

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Callable, Optional, Union
```

Comment:
I checked these types by looking at the c source code and running mypy against the `test` module with the following setup:


```
; mypy.ini
[mypy]
show_column_numbers=True
pretty=False
show_error_codes = True

disallow_any_unimported=True
disallow_any_expr=False
disallow_any_decorated=True
disallow_any_explicit=False
disallow_any_generics=True
disallow_subclassing_any=True

disallow_untyped_calls=True
disallow_untyped_defs=True
disallow_incomplete_defs=True
check_untyped_defs=True
disallow_untyped_decorators=True

no_implicit_optional=True
strict_optional=True

warn_redundant_casts=True
warn_unused_ignores=True
warn_no_return=True
warn_return_any=False
warn_unreachable=True

strict_equality=True

ignore_missing_imports=False
```

```
./.venv/bin/mypy test
```

Reply:
https://docs.python.org/3/c-api/arg.html was helpful for figuring out the arg types